src/tiled/propertybrowser.cpp

```diff
@@ -494,6 +494,24 @@ void PropertyBrowser::addTilesetProperties()
     QtProperty *groupProperty = mGroupManager->addProperty(tr("Tileset"));
     createProperty(NameProperty, QVariant::String, tr("Name"), groupProperty);
     createProperty(TileOffsetProperty, QVariant::Point, tr("Drawing Offset"), groupProperty);
+
+    // Next properties we should add only for non 'Collection of Images' tilesets
+    if (mObject->typeId() != Object::TilesetType)
+        return;
```

Comment:
Wait a second, this whole check should be removed. In the `addTilesetProperties` function, you can already safely assume that the `mObject` is a `Tileset` instance.